whois-update/src/test/java/net/ripe/db/whois/update/authentication/strategy/MntByAuthenticationTest.java

```diff
@@ -66,6 +68,8 @@ public void supports_every_object_with_a_mntby_attribute() {
 
         when(update.getType()).thenReturn(ObjectType.INETNUM);
         assertThat(subject.supports(update), is(true));
+
+        verifyZeroInteractions(maintainers);
```

Comment:
I guess, that is a different point. I.E. cleaning up or improving the organisation of tests.